components/event-footer/EventFooter.vue

```diff
@@ -56,6 +70,7 @@
   }
 
   .event-footer__link {
+    z-index: 1;
```

Comment:
Does this make a difference? Couldn't see anything change, one `z-index: -1` seems clearer to me?

Reply:
It made the image disappear.